dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -2104,14 +2104,31 @@
                  and '%(ProjectReferenceWithConfiguration.ReferenceOutputAssembly)' == 'true'"
       SkipNonexistentTargets="true"
       ContinueOnError="$(_ContinueOnError)">
-      <Output TaskParameter="TargetOutputs" ItemName="_PackagingOutputsFromOtherProjects"/>
+      <Output TaskParameter="TargetOutputs" ItemName="_PackagingOutputsFromOtherMrtCoreProjects"/>
+    </MSBuild>
+
+    <!-- The referenced project may not be using MRTCore. In that case, GetMrtPackagingOutputs won't be defined. However, if the
+         project is of type UWP, it'll have GetPackagingOutputs defined. So, try calling GetPackagingOutputs. If it does not exist,
+         we simply no-op - see the SkipNonexistentTargets parameter below. -->
+    <MSBuild
+      Projects="@(ProjectReferenceWithConfiguration)"
+      Targets="GetPackagingOutputs"
+      BuildInParallel="$(BuildInParallel)"
+      Properties="%(ProjectReferenceWithConfiguration.SetConfiguration); %(ProjectReferenceWithConfiguration.SetPlatform)"
+      Condition="'@(ProjectReferenceWithConfiguration)' != ''
```

Comment:
You should condition this so it doesn't get run if the Mrt target is run, in the case of UWP + Mrt.Pri nupkg reference

Reply:
Currently, this combination is not possible. WinUI 3.0, though usable in a UWP, won't import MRTCore. When used directly, MRTCore will noop in a UWP. Supposing this were possible though, yes, both targets will be defined. In that case, there's no harm, is there? We'll just get duplicate entries which will then be de-duped? We should still do this, just trying to understand the situation if we don't.

Reply:
My thinking is that we know that the targets are intended for mutually exclusive scenarios so it's safer to enforce that (by using a condition to ensure that the second is run only if the first did not run i.e. yielded no results) rather than have an implicit dependency on some later step. Additionally, running the second target unnecessarily results in extra work that then slows down the build.

Reply:
Talked offline. Decision was made to defer addressing this currently-disabled scenario.